packages/story-editor/src/components/footer/carousel/useCarouselScroll.js

```diff
@@ -97,9 +107,19 @@ function useCarouselScroll({
     [pageThumbWidth, pageThumbMargin]
   );
 
+  // Is this the first scroll (which will be instant rather than animated)?
+  const firstScroll = useRef(true);
+
+  // If the carousel drawer is collapsed, reset first scroll to true
+  useEffect(() => {
+    if (carouselState == CAROUSEL_STATE.CLOSED) {
```

Comment:
```suggestion
    if (carouselState === CAROUSEL_STATE.CLOSED) {
```

Reply:
Completely missed this review! Fixed in fb365f14c9.